app/gateway/traditional-mode.md

```diff
@@ -63,6 +63,18 @@ style D stroke:none,fill:#0E44A2,color:#fff
 > _Figure 1: In a traditional deployment, all {{site.base_gateway}} nodes connect to the database. 
 Each node manages its own configuration._
 
+## About {{site.base_gateway}} clusters
+
+A {{site.base_gateway}} cluster does not load balance client traffic across Data Plane nodes out-of-the-box. You still need a
+[load-balancer](/gateway/load-balancing/) in front of your Data Plane nodes to distribute your traffic. Instead,
+a {{site.base_gateway}} cluster means that those nodes will share the same configuration.
```

Comment:
```suggestion
Data Plane nodes in the same {{site.base_gateway}} cluster share the same configuration.

To load balance traffic across Data Plane nodes in a cluster, configure a [load balancer](/gateway/load-balancing/) in front of your nodes to distribute the traffic. 
```
This feels like a more natural order to these facts.
Not sure that you need to say "gateway doesn't load balance traffic out of the box"; it's generally better to phrase it in a way of what you _can_ do, ie in this case "hey look, you can load balance across nodes, go configure it".